tsconfig.base.json

```diff
@@ -21,15 +21,21 @@
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "paths": {
+      "@bigcommerce/checkout-js/apple-pay-integration": [
```

Comment:
Maybe one for later, we discussed package names for SDK. Maybe we need to apply similar logic here?

Reply:
I think we end up settling with the `/` because these packages are never meant to be published via npm.

On a separate note, the parent package should be `@bigcommerce/checkout` rather than `@bigcommerce/checkout-js`. We rename all of them together in a separate PR.